strongbox-data-service/pom.xml

```diff
@@ -249,9 +237,19 @@
         </dependency>
 
         <dependency>
-            <groupId>com.esotericsoftware</groupId>
-            <artifactId>kryo</artifactId>
-            <version>5.0.0-RC1</version>
+            <groupId>org.springframework.data</groupId>
+            <artifactId>spring-data-neo4j</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.neo4j</groupId>
+                    <artifactId>neo4j-ogm-bolt-driver</artifactId>                    
+                </exclusion>
+            </exclusions>
+        </dependency>
```

Comment:
Just for my better understanding, what issue what this excluded dependency causing, (or do we just not need it)?

Reply:
We don't need it. This driver is for `neo4j` and we using our own custom driver which is based on `cypher-for-gremlin`.